### PR TITLE
VSL-22: hook up address copying and name editing from safe settings

### PR DESCRIPTION
**Pull request by jakemmarsh**
- Added `border-light` to some inputs that weren't matching designs
- Refactored `useClipboard` hook to return a `textJustCopied` value instead of a boolean so that it can be used for multiple values
- Hooked up "Copy" for address in safe settings
- Added new `<EditSafeName />` modal component, hooked up to "Edit" in safe settings

https://user-images.githubusercontent.com/2916671/172436257-5193e30b-f6b7-49c6-a7d3-3b444af3d03c.mov

### CODE REVIEW

File: packages/client/src/hooks/useTreasury.js [hunk 1]
```diff
@@ -279,6 +279,16 @@ export default function useTreasury(treasuryAddr, userAddr) {
...
   };
 
+  const renameTreasury = (treasuryAddr, newName) => {
+    dispatch({
+      type: "RENAME_TREASURY",
+      payload: {
+        address: treasuryAddr,
+        name: newName,
+      },
+    });
```
Comment (mannynotfound) [inline packages/client/src/hooks/useTreasury.js:289]: I made `SET_TREASURY` a bit generic on purpose so you could be able to just call

```
dispatch({
      type: "SET_TREASURY",
      payload: {
        [address]: {
          name: newName,
        }
    });
```

File: packages/client/src/pages/Safe.js [hunk 2]
```diff
@@ -281,29 +283,31 @@ function Safe({ web3 }) {
...
     );
   };
 
   const onReceive = () => {
     modalContext.openModal(
-      React.createElement(ReceiveTokens, {
-        name: safeData.name,
-        address,
-      })
+      <ReceiveTokens name={safeData.name} address={address} />
```
Comment (mannynotfound) [inline packages/client/src/pages/Safe.js:310]: lord knows why I didnt write this in jsx originally but thank you

**Review by mannynotfound — COMMENTED** [review]
LGTM I think we can just remove redundant reducer action

File: packages/client/src/hooks/useTreasury.js (hunk 1 above)
Comment (jakemmarsh) [reply packages/client/src/hooks/useTreasury.js]: ah makes sense, removed new action and used this instead

**Pull request merged**